pkg/scheduler/metrics/metrics.go

```diff
@@ -83,15 +83,15 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
-	PreemptionVictims = metrics.NewHistogram(
+	PreemptionVictims = metrics.NewHistogramVec(
 		&metrics.HistogramOpts{
 			Subsystem: SchedulerSubsystem,
 			Name:      "preemption_victims",
-			Help:      "Number of selected preemption victims",
+			Help:      "Number of selected preemption victims, by the priority of the preemptor pod.",
 			// we think #victims>64 is pretty rare, therefore [64, +Inf) is considered a single bucket.
 			Buckets:        metrics.ExponentialBuckets(1, 2, 7),
 			StabilityLevel: metrics.STABLE,
-		})
+		}, []string{"priority"})
```

Comment:
This is a STABLE metric, I think this is not acceptable, right? @logicalhan

Reply:
Oh, sorry, I didn't know that. Is there relevant documentation? Can't I just add a tag?

Reply:
I believe @kerthcet is correct.  From https://kubernetes.io/docs/reference/instrumentation/metrics/#list-of-stable-kubernetes-metrics:

> Stable metrics observe strict API contracts and no labels can be added or removed from stable metrics during their lifetime.

Reply:
I completely didn't know that, good to know.

Reply:
Thanks for your guide 🙏 Is adding the new metrics the only way to achieve https://github.com/kubernetes/kubernetes/issues/122046?

Reply:
Yes, and I'm honestly not sure it deserves creating a new metric.
@kerthcet What do you think.

Reply:
Yes, keeping two similar metrics is not a good idea. Maybe we can back to this once we got similar feedbacks from the community.

Reply:
As a user in production operations, it is a label I want.

Reply:
+1 for @kerthcet.
I agree that it's valuable though, let's once close this PR while keeping https://github.com/kubernetes/kubernetes/issues/122046.
If we've got the same feedback, or we've got another feature request for this metric, then we can consider creating a new one to eventually replace this metric.